test/kubemark/start-kubemark.sh

```diff
@@ -21,6 +21,8 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
+export ENABLE_APISERVER_INSECURE_PORT="${ENABLE_APISERVER_INSECURE_PORT:-false}"
```

Comment:
this should be KUBE_ENABLE_APISERVER_INSECURE_PORT

Reply:
outdated since the ENABLE_APISERVER_INSECURE_PORT is removed.